File: presets/MidiKit/Lua/Filter Ch2.lua
--[[
@target stoermelder MIDI-KIT
@engine minilua@v1
@author stoermelder
@description Drops all incoming MIDI messages except for MIDI channel 2
--]]

midi.onMessage = function(midiPort, msg)
    if midi.getChannel(msg) == 2 then
        midiOut.send(msg)
    end
end

File: presets/MidiKit/JavaScript/Filter Ch2.js
/**
 * @target stoermelder MIDI-KIT
 * @engine QuickJs@v1
 * @author stoermelder
 * @description Drops all incoming MIDI messages except for MIDI channel 2
 */

rack.onMidiMessage = function(midiPort, msg) {
    if (midi.getChannel(msg) === 2) {
        midiOut.send(msg);
    }
}
